atomicapp/providers/docker.py

```diff
@@ -90,6 +90,8 @@ def run(self):
             label_run = None
             with open(artifact_path, "r") as fp:
                 label_run = fp.read().strip()
+                # if docker-run provided as multiline command
+                label_run = ' '.join(label_run.split('\\\n'))
```

Comment:
@cdrage there are no multiple lines its a single line, backslashes make it single line